linting-requirements.txt

```diff
@@ -0,0 +1 @@
+ruff
```

Comment:
This one is not in the workflow. I would rather have it in the test-requirements to avoid the proliferation of requirements.